test/TestExtensions/DefaultClusterFixture.cs

```diff
@@ -8,8 +8,11 @@ public class DefaultClusterFixture : BaseTestClusterFixture
         protected override TestCluster CreateTestCluster()
         {
             var options = new TestClusterOptions();
+#if !NETSTANDARD_TODO
+            // waiting for OrleansProviders project to be ported over to vNext
             options.ClusterConfiguration.AddMemoryStorageProvider("Default");
             options.ClusterConfiguration.AddMemoryStorageProvider("MemoryStore");
+#endif
```

Comment:
I'm not sure git is smart emough to recognise this . this may still appear as different diff to it. We shouldn't need to do this in the future. The merge conflict is not hard to deal with.